packages/v4/patternfly-docs/content/design-guidelines/components/notification-drawer/notification-drawer.md

```diff
@@ -49,25 +49,15 @@ If you want to sort notifications by category, collapsible headers may be used t
     - **Unclear last**: Unclears that last cleared or set of cleared notifications (if supported).
     - **Settings**: Opens notification settings.
 
-### Placement in a page
-
-<img src="./img/Notification-drawer-in-page.png" alt="notification drawer placement" width="825"/>
-
-
-1. **Notification badge:** Whenever the notification drawer is used, a [notification badge](/components/notification-badge) must be placed in the masthead. Clicking on the badge will toggle the drawer open or closed. Depending on options supported in the application, the badge can also be used to display the number of notifications in the drawer and whether there are unread or critical notifications needing attention. See the [notification badge](/components/notification-badge) usage guidelines for more information about these options.
-
-2. **Drawer:** The notification drawer lives in slide-out panel that overlays page content and is anchored to the right-hand edge of the viewport.
-
-
 ## Usage
```

Comment:
Maybe don't need to address in this PR - but the font in the last image of these guidelines looks really weird, could you make a separate issue to fix that?